Makefile

```diff
@@ -0,0 +1,27 @@
+.PHONY: codegen web3_codegen orderbook_codegen subgraph_codegen test lint format remove_unused_imports
+
+codegen: web3_codegen orderbook_codegen subgraph_codegen
+
+# web3_codegen:
+# 	poetry run web3_codegen
```

Comment:
Is this covered in subsequent PRs, or should this comment be removed?

Reply:
not relevant anymore 👍